gemma-web/src/main/java/ubic/gemma/web/controller/OntologyController.java

```diff
@@ -98,7 +128,7 @@ public String getFactorValue( @PathVariable("factorValueId") Long factorValueId,
         StringBuilder s = new StringBuilder();
         s.append( String.format( "<title>FactorValue #%d: %s</title>", factorValueId, escapeHtml4( oi.getLabel() ) ) );
         s.append( "<div class=\"padded\">" );
-        s.append( String.format( "<h2>FactorValue #%d: %s</h2>", factorValueId, renderOntologyResource( oi ) ) );
+        s.append( String.format( "<h2 style=\"padding-bottom:0.5em\">FactorValue #%d: %s</h2>", factorValueId, renderOntologyResource( oi ) ) );
```

Comment:
There are pre-defined spacing classes in `spacing.css`. You should use those instead. I don't think I have a class for 0.5em, but you can add one.